prediction_market_agent/agents/microchain_agent/omen_functions.py

```diff
@@ -0,0 +1,23 @@
+from microchain import Function
+from prediction_market_agent_tooling.markets.omen.omen import (
+    redeem_from_all_user_positions,
+)
+
+from prediction_market_agent.agents.microchain_agent.utils import MicrochainAPIKeys
+
+
+class RedeemWinningBets(Function):
+    @property
+    def description(self) -> str:
+        return "Use this function to redeem winnings from a position that you opened which has already been resolved. Use this to retrieve funds from a bet you placed in a market, after the market has been resolved."
```

Comment:
Maybe add something like "If you have outstanding winnings to be redeemed, your balance will be updated".

Reply:
added, thanks